### ISSUE TITLE
Training with large scale data doesn't work.

### ISSUE THREAD

**Issue by yusuke-ai** [winddori2002/DEX-TTS#6]
@winddori2002 

Thank you for providing awesome work!
I'm training the DEX-TTS model.
The issue I encountered is that the training loss won't decrease well and the output voice quality is bad (robotic sound) with 3000 hours dataset. With the original lr, the loss easily become nan, with smaller lr, the loss decreases but the loss doesn't drop enough.

The model works with 1 hours dataset, and around 100 hours dataset, but didn't work with 3000 hours dataset.

Do you think that the model can't be trained with more than 1000 hours dataset or 
is is just because of the hyperparameter or training configuration like optimizer?

Could you give me some advice?

Thank you!

**Comment by winddori2002** [comment]
Hi, thanks for your interest.

I've not tried training the models with more than 1,000 hours dataset, but I tried LibriTTS (about 500 hours).
I uploaded the yaml file in the DEX-TTS/config/LibriTTS/base.yaml (the model size is about 39M).
I think the model should be bigger for training 3000 hours dataset. 
You may increase the dimension size of the encoder, decoder, and dit dimension or layers.
The dimension size of tv_encoder, lf0_encoder, tiv_encoder should be analyzed (I tried up to 384)

**Comment by yusuke-ai** [comment]
Thank you!

I will try training using your configuration for LibriTTS!

**Comment by winddori2002** [comment]
I think that the model may require more params considering the dataset size (3000 h)

**Comment by yusuke-ai** [comment]
@winddori2002 
Thank you!

I tried with the same network structure you gave me. (increased the various dimensions)
It become nan after around 10 hours... with 2 gpus...

Also I tried to increase depth of decoder and encoder, but it started to diverge after 2 days with 2 gpus...

Do you have any other ideas?

I will try to train the model with the reduced dataset of 500 hours next.

**Comment by winddori2002** [comment]
Can you check if the AMP is used? 
Usually, using AMP may cause nan loss.

**Comment by yusuke-ai** [comment]
@winddori2002 
Oh. I see. Probably I should turn off AMP like in your configuration.
I will try with AMP turned off.

Can I confirm one more thing?
In my training, Duration loss is around 0.66 and the prior loss is around 1.45 even if I tried many model hyperparameters and training configurations. Is it normal? What was the loss when you trained LibriTTS and VCKT?

Thank you!

**Comment by winddori2002** [comment]
In my case, 
VCTK: duration loss (0.3), prior loss (1.1)
LibriTTS-train-clean(only): duration loss (0.39), prior loss (1.2)

Duration loss tends to decrease more easily than prior loss while training. 
However, prior or diffusion loss does not dramatically decrease after converging once.

Thanks

**Comment by yusuke-ai** [comment]
I see, then something is wrong in my training,
because the duration loss is still high(0.66) after converging.
Also it's same when AMP is turned off (I experimented it in the past)

Anyway, I will try with subset dataset and gradually increase the dataset size.
